src/js/ctrl/export.ts

```diff
@@ -47,19 +51,20 @@ export function exporter(internal: boolean = false): Spec {
   spec.data = exporter.pipelines(state, int);
 
   // Add interactions and widgets from store
-  spec = exporter.interactions(state, spec);
+  spec = history ? spec : exporter.interactions(state, spec);
   spec = exporter.widgets(state, spec);
   return spec;
 }
 
 exporter.interactions = function(state: State, spec) {
-  state.getIn(['vis', 'present', 'interactions']).forEach((interaction: InteractionRecord) => {
-    const group: GroupRecord = state.getIn(['vis', 'present', 'marks', String(interaction.groupId)]);
+  let storeState = store.getState(); // use Store state instead
```

Comment:
same comment as above

Reply:
not sure exactly what this reference but I think I resolve it in my next PR. Lmk if this is still an issue.